### DISCUSSION TITLE
Is there a short command for block all countries?

### DISCUSSION THREAD

**Discussion by Jaipharma** [dvershinin/fds#28] [Ideas]
I would like to block incoming from all countries then allow incoming only from my counties.
Is there a short command for this?

**Comment by dvershinin** [comment]
None as of yet. 

This would require  setting default zone to "drop" and adding another FirewallD zone for trusted countries which which has priority  over drop, e.g. 000-trusted. 

Pull requests are welcome as I don't have much time for now.
